dali/dfuplus/dfuplus.cpp

```diff
@@ -64,7 +64,7 @@ class CDafsThread: public Thread
         else
             listenep.getUrlStr(eps);
         enableDafsAuthentication(requireauthenticate);
-        server.setown(createRemoteFileServer());
+        server.setown(createRemoteFileServer()); // no throttle limitting
```

Comment:
I meant it to use none here, hence the comment.
I think I change the prototype to use defaults but not this call.